src/scripts/services/dimStoreService.factory.js

```diff
@@ -21,13 +21,28 @@ function StoreService(
   dimManifestService,
   $translate,
   uuid2,
-  dimFeatureFlags
+  dimFeatureFlags,
+  dimSettingsService,
+  $http
 ) {
   var _stores = [];
   var _idTracker = {};
 
   var _removedNewItems = new Set();
 
+  var dimClassifiedData;
+  var dimClassifiedDataPromise;
+
+  if ($DIM_FLAVOR === 'dev') {
+    dimClassifiedData = require('app/data/classified.json');
+  } else {
+    dimClassifiedDataPromise = $http.get('https://beta.destinyitemmanager.com/data/classified.json')
```

Comment:
Rather than loading this right up front, it should probably be kicked off as part of the store load